contrib/sync_namespaces_jobs.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+import argparse
+import os
+
+import yaml
+
+DST_DIR = "/tmp/tron-servdir"
```

Comment:
Let's make destination and source arguments as well. This will make it easier to use, especially when we move configs out of tron/prod.